### ISSUE TITLE
Option to view mouse coordinates 

### ISSUE THREAD

**Issue by Unknow0059** [d2phap/ImageGlass#612]
### Describe the solution you'd like
To be able to view the mouse coordinates by default.

### Describe alternatives you've considered
I can use the Color Picker Tool to view the mouse coordinates, but that's kinda annoying. 
I would like for the option to be toggleable in the settings, and for the coordinates to be always visible in an area of the UI that is non-intrusive to the image viewing.

**Comment by johmarjac** [comment]
Yes please, i need this too!

**Comment by zhou-wb** [comment]
Yes! And not only show the mouse coordinates, but also show the color on that coordinate! Thank you!

**Comment by d2phap** [comment]
@zhou-wb 
> but also show the color on that coordinate!

You can use Color Picker tool to pick color

**Comment by zhou-wb** [comment]
> @zhou-wb
> 
> > but also show the color on that coordinate!
> 
> You can use Color Picker tool to pick color

Yes, but it's better if the color picker can always show the color without click
